model-engine/model_engine_server/api/app.py

```diff
@@ -91,6 +109,7 @@ def load_redis():
     get_or_create_aioredis_pool()
 
 
+# these routes should match those exempt from the concurrency limiter in the middleware
```

Comment:
To better link the routes between here <> middleware with code, we could define a shared list variable. Something like this could work. 

```
health_routes = ["/healthcheck", "/healthz", "/readyz"]

def healthcheck() -> Response:
    """Returns 200 if the app is healthy."""
    return Response(status_code=200)

for endpoint in health_routes:
    app.get(endpoint)(healthcheck)
```